stubstub/stub.js

```diff
@@ -4,28 +4,45 @@ import sinon from 'sinon';
 import { getEmptyDoc } from 'components/Editor';
 import * as firebaseAdmin from '../server/utils/firebaseAdmin';
 
-export const stubModule = (module, functionNames) => {
-	if (!Array.isArray(functionNames)) {
-		// eslint-disable-next-line no-param-reassign
-		functionNames = [functionNames];
+const stubManyFunctions = (module, stubEntries) =>
```

Comment:
None of this is central to the purpose of the PR — just improving the ergonomics of `stubOut` so you can pass it a `name: function` mapping as well as a single function name or list of function names, and a module path rather than a `required` module.